api/dbadapters/bigquery.ts

```diff
@@ -27,33 +27,32 @@ export class BigQueryDbAdapter implements DbAdapter {
   }
 
   execute(statement: string) {
-    return Bluebird.resolve().then(() =>
+    return new Bluebird<any[]>((resolve, reject, onCancel) => {
+      let isCancelled = false;
+      const eEmitter = new EventEmitter();
+
+      onCancel(() => {
+        isCancelled = true;
+        eEmitter.emit("jobCancel");
```

Comment:
pull out this constant "jobCancel" string since want to ensure the same value is used everywhere for this event?